client/extensions/woocommerce/state/sites/settings/general/selectors.js

```diff
@@ -111,3 +111,27 @@ export const areTaxCalculationsEnabled = ( state, siteId = getSelectedSiteId( st
 	}
 	return 'yes' === taxesEnabled.value;
 };
+
+export const isStoreNoticeEnabled = ( state, siteId = getSelectedSiteId( state ) ) => {
+	const generalSettings = getRawGeneralSettings( state, siteId );
+	const noticeEnabled = find( generalSettings, { id: 'woocommerce_demo_store' } );
+	if ( ! noticeEnabled ) {
+		return false;
+	}
+	if ( ! ( 'value' in noticeEnabled ) ) {
+		return false;
+	}
+	return 'yes' === noticeEnabled.value;
+};
+
+export const getStoreNotice = ( state, siteId = getSelectedSiteId( state ) ) => {
+	const generalSettings = getRawGeneralSettings( state, siteId );
+	const notice = find( generalSettings, { id: 'woocommerce_demo_store_notice' } );
```

Comment:
naive question here: How come hard coded ids `woocommerce_demo_store_notice` and `woocommerce_demo_store`? Are these simply demo values?

Reply:
Those are the actual keys from the settings-general REST API elements - their names (i.e. whether this is a demo store or not) aren't quite in line with how they ended up being used over the years (as a means of presenting any notice to customers).